terraform/digitalocean/vpc/README.md

```diff
@@ -0,0 +1,28 @@
+# VPC
+
+Module creates or re-use existing VPC in DigitalOcean.
+
+If vpc_id =
+  "create" - creates VPC
+  "default" - use default VPC in provided region
+  "vpc_id"  - it would use provided id
+}
```

Comment:
[p2] Now it's one big string. You mean next:
```suggestion
If `vpc_id` equal to:
* `create` - creates VPC
* `default` - use default VPC in provided region
* `vpc_id`  - it would use provided id
```

Reply:
Fixed